assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -360,6 +361,7 @@ function MediaPane(props) {
         ) : (
           <PaginatedMediaGallery
             providerType="local"
+            hasUploadMediaAction={hasUploadMediaAction}
```

Comment:
Thoughts on renaming this prop everywhere to something more concise?

```suggestion
            canEditMedia={hasUploadMediaAction}
```

Reply:
Yeah, works for me.